### ISSUE TITLE
Replace link to docs for snapping

### ISSUE THREAD

**Issue by tomasMizera** [MerginMaps/qgis-mergin-plugin#460]
The _documentation_ link here:

<img width="708" alt="image" src="https://user-images.githubusercontent.com/22449698/192960414-52e5e658-f38f-4d97-b572-093cfd6fd9ba.png">

Takes you to https://merginmaps.com/docs/gis/features/#project-settings. While it has some words about snapping, there is an entire article dedicated to snapping: https://merginmaps.com/docs/gis/snapping/. We should rather redirect users directly to the latter one.

TL;DR Change link from `https://merginmaps.com/docs/gis/features/#project-settings` to `https://merginmaps.com/docs/gis/snapping/`
